app/views/patients/show.html.erb

```diff
@@ -61,48 +61,57 @@
   </div>
 </div>
 
-<%= link_to "Add Visit Details", new_district_facility_patient_visit_path(@district, @facility, @patient), class: "btn btn-primary" %>
-
-<table class="table table-hover">
-  <thead>
-    <tr>
-      <th>Date Attended</th>
-      <th>Blood Pressure</th>
-      <th>Blood Sugar</th>
-      <th>Medications</th>
-      <th>Next Visit</th>
-      <th>
-    </tr>
-  </thead>
-  <tbody>
+<h4>Visits</h4>
+<div class="row mb-5">
+  <% if @patient.visits.any? %>
+    <div class="col-md-2">
+      <span class="visit-info visit-header">Date Attended</span>
+      <span class="visit-info visit-header">Systolic</span>
+      <span class="visit-info visit-header">Diastolic</span>
+      <span class="visit-info visit-header">Blood Sugar</span>
+
+      <% %i(amlodipine telmisartan enalpril chlorthalidone aspirin statin beta_blocker).each do |drug| %>
+        <span class="visit-info visit-header"><%= drug.to_s.humanize %></span>
+      <% end %>
+
+      <% (1..3).each do |drug_number| %>
+        <span class="visit-info visit-header">Medication <%= drug_number %></span>
+      <% end %>
+
+      <span class="visit-info visit-header">Referred to Specialist</span>
+      <span class="visit-info visit-header">Date of Next Visit</span>
+    </div>
+
     <% @patient.visits.order(:measured_on).each do |visit| %>
```

Comment:
The columns don't wrap nicely. We should allow for horizontal scrolling, instead of overflowing vertically. Or maybe we could batch the columns.  [
![screenshot 2018-11-21 at 15 37 10](https://user-images.githubusercontent.com/3933133/48834368-a7795900-eda3-11e8-8f03-b4f4ea2368cf.png)
](url)

Reply:
I've fixed this now (by batching columns)